db/migration/1647947484386-UpdateCharaterSet.ts

```diff
@@ -0,0 +1,47 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class UpdateCharaterSet1647947484386 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        // Note that as a first step we change the variables name column from 1000 chars long to 750 so that
+        // once we use the potentially 4 byte UTF8mb4 encoding the unique index will still stay below the 3072 byte limit
+        const queries = `SET FOREIGN_KEY_CHECKS = 0;
+ALTER TABLE \`grapher\`.\`variables\` modify column name VARCHAR(750);
```

Comment:
Was just wondering what length our longest names are, apparently these (429 chars):

| id | name |
|-|-|
| 150649 | 16.1.2 - Number of conflict-related deaths (civilians), by sex, age and cause of death (Number) - VC_DTH_TOCVN - All age ranges or no breaks by age - Both sexes - Incendiary; Chemical, biological, radiological, nuclear (CBRN); Electromagnetic weapons; Less lethal weapons; Denial of access to/ destruction of objects indispensable to survival; Accidents related to conflict; Use of objects and other means; Multiple Weapons Used. |
| 179312 | 16.1.2 - Number of conflict-related deaths (civilians), by sex, age and cause of death (Number) - VC_DTH_TOCVN - All age ranges or no breaks by age - Both sexes - Incendiary; Chemical, biological, radiological, nuclear (CBRN); Electromagnetic weapons; Less lethal weapons; Denial of access to/ destruction of objects indispensable to survival; Accidents related to conflict; Use of objects and other means; Multiple Weapons Used. |
| 312587 | 16.1.2 - Number of conflict-related deaths (civilians), by sex, age and cause of death (Number) - VC_DTH_TOCVN - All age ranges or no breaks by age - Both sexes - Incendiary; Chemical, biological, radiological, nuclear (CBRN); Electromagnetic weapons; Less lethal weapons; Denial of access to/ destruction of objects indispensable to survival; Accidents related to conflict; Use of objects and other means; Multiple Weapons Used. |